### PR TITLE
 Improvement - 337 - Small fix: change color of the Android status bar

### PR DESCRIPTION
**Pull request by spasma**

### CODE REVIEW

File: src/js/routes.js [hunk 1]
```diff
@@ -1251,10 +1251,8 @@ angular.module('copayApp').config(function(historicLogProvider, $provide, $logPr
       if (screen.width < 768 && platformInfo.isCordova)
         screen.lockOrientation('portrait');
 
-      if (window.StatusBar) {
-        if (ionic.Platform.isAndroid()) {
+      if (window.StatusBar && ionic.Platform.isAndroid()) {
```
Comment (jbdtky) [inline src/js/routes.js]: Consistent StatusBar

**Comment by spasma** [comment]
@jbdtky With "window." removed, the app breaks while running in the browser and desktop. I also think it doesn't work on devices, but I cannot confirm at this moment

**Comment by jbdtky** [comment]
I tested on device and it works. I will try on my browser.

**Comment by jbdtky** [comment]
Anyway, we should first test if it is a android device then test if StatusBar. I will make the change now.

**Pull request merged**